### DISCUSSION TITLE
生医方向博士生论文要求参考文献为Cell期刊格式，可以这么改一改

### DISCUSSION THREAD

**Discussion by yonggeli66** [tuna/thuthesis#842] [Ideas]
（thu的）生医方向博士生论文要求参考文献为Cell期刊格式，可以参考这个链接添加bst文件：
https://www.bibtex.com/s/bibliography-style-cell-cell/#download%EF%BC%8C

PS：如果有文献在bib中没有写卷宗（volume）可能会导致编译报错

**Comment by LUKE-ZZ** [comment]
\bibliographystyle{cell}
如果是用Overleaf的话，直接加上这行命令就可以了，其他LaTex编译环境不知道是否可行

**Comment by zepinglee** [comment]
这个 `cell.bst` 比较古老了，而且在我印象中有很多 bug，跟模板调用的 `natbib` 有一些兼容性问题。

**Comment by yonggeli66** [comment]
感谢解答！后面的同学可以参考~！

**Comment by zepinglee** [comment]
<https://www.cell.com/news-do/cell-press-referencing-style>:

> Cell Press is moving away from the Harvard referencing style (e.g., Smith et al., 2020), the standard for our life science journals since Cell launched in 1974. We ask all new and revised submissions after October 2022 to format citations using the superscript numbered Cell Press referencing style (e.g., “Multiple reports support this observation.1,2”).

你们需要使用新的样式吗？

**Comment by LUKE-ZZ** [comment]
> 

你好，我是生命学院的，我们今年9月预答辩需要提交的论文初稿中，仍然按照旧板的cell参考文献格式，是否会在之后修改为新版还未知。

**Comment by yonggeli66** [comment]
> 
目前没有说要改，正文里的引用还是人名+时间(Smith et al., 2020)的形式（\citep{}）

**Comment by niyunsheng** [comment]
请问，这个问题现在怎么解决呢？用overleaf添加cell格式期刊引用错误。如下图显示的括号不是在作者前面

![image](https://github.com/tuna/thuthesis/assets/62385270/5547b139-78f8-4e16-ace6-e158c1123c6b)
